### ISSUE TITLE
Windows executable detected as Malware by several Antivirus engines

### ISSUE THREAD

**Issue by piraces** [achannarasappa/ticker#3]
I was trying to try the app in a Windows environment when the browser started to cancel the download of the compressed file...
However, I managed to download it and execute the program, which resulted in the result:

```
❯ ticker

Unhandled Exception: System.ComponentModel.Win32Exception: Operation did not complete successfully because the file contains a virus or potentially unwanted software
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at shim.CommandExecutor.execute(String process, String arguments, String working_directory, Boolean is_gui, Boolean wait_for_exit, Boolean requires_elevation)
   at shim.ShimProgram.Main(String[] args)
```

Then I uploaded the file to VirusTotal, where seven Antivirus engines interpreted it as Malware/Malicious (including Microsoft itself). You can see the report here: https://www.virustotal.com/gui/file/6a89d65b6ffab900b0a1d66ad1b49b73d8a0aa147a6a08524c4af34c6cc4626f/detection

I want to think that this is not intended, but I don't know why this is happening...

**Comment by achannarasappa** [comment]
Thanks for reporting this. I am not that familiar with how these virus scanners work but I'll do a bit of research and come back with some more information on why they could be flagging this as malware.

**Comment by achannarasappa** [comment]
I've run two additional scans against the 32bit and 64bit versions of the latest release. I've found that both were flagged by different scanners but both were flagged as potential malware.

Below are the scan results:
https://www.virustotal.com/gui/file/442c0743bb817ef27187e4f77746c966cf6684fdf59a92759d5f4734f4b72c79/detection
https://www.virustotal.com/gui/file/5970c17d0e6b80a7e65d9aa43d4b5dd8ddedd71106ff6407ae43a2da4f695c5c/detection

As for the reason why these are flagged a malware, I found [this article](https://www.csoonline.com/article/3216765/heres-why-the-scanners-on-virustotal-flagged-hello-world-as-harmful.html) that explains even some hello world programs and new releases from major software groups (e.g. Mozilla Firefox) are flagged as malware/viruses.

I am not sure how to resolve the above detections but it may require reaching out to scanner companies and requesting `ticker` be added to their list of false positives.

In lieu of a clean scan, I'd invite you to review the source of the application here on GitHub to validate yourself there is no malicious code present. Releases here on GitHub are also built and deployed through GitHub Actions such that releases are not produced directly from my machine but rather on GitHub will full transparency into with which code and build steps produce the release.

I understand your concerns with the scans not coming back clean and if you choose not to use ticker for that reason but I hope the above provides some context as to why this is happening.
